### ISSUE TITLE
wrapper_styles undefined for vimeo video

### ISSUE THREAD

**Issue by WHITE-developer** [stamat/youtube-background#42]
Uncaught ReferenceError: wrapper_styles is not defined `jquery.youtube-background.js:562`

**Comment by awavevladimirmironov** [comment]
this is not fixed yet?

**Comment by Finn84FIN** [comment]
There seems to be this extra section which is not needed in the `VimeoBackground.prototype.injectPlayer` function. wrapper_styles is not defined in that function yet. Commenting that part out seems to have fixed the issue. 

```
if (this.params['load-background'] || this.params['poster']) {
      //if (this.params['load-background']) wrapper_styles['background-image'] = 'url(https://img.youtube.com/vi/'+this.ytid+'/maxresdefault.jpg)';
      if (this.params['poster']) wrapper_styles['background-image'] = this.params['poster'];
      wrapper_styles['background-size'] = 'cover';
      wrapper_styles['background-repeat'] = 'no-repeat';
      wrapper_styles['background-position'] = 'center';
    }
```

**Comment by stamat** [comment]
Seems like this part is an overlooked code from some older versions, this is fixed now in **v1.0.18**

Sorry for the inconvenience folks @Finn84FIN @WHITE-developer @awavevladimirmironov
